src/Controller.cpp

```diff
@@ -122,6 +145,15 @@ void Controller::indicateMode(byte mode)
     _ui->indicateMode(mode);
   }
   
+  if ((mode == MODE_UNINITIALISED) || (mode == MODE_SETUP))
+  {
+    bitClear(_mparams[8], 2);
```

Comment:
I would use PF_FLIM constant here instead of a literal '2'. 
(Yes, I will add a PF_NORMAL to replace PF_FLIM.)
Then replace the bitClear with "_params[8] &= PF_FLIM;

Reply:
Yes, much better.  It then gets rid of the Arduino macros